Lib/fontTools/cu2qu/ufo.py

```diff
@@ -91,7 +92,11 @@ def closePath(self):
     def endPath(self):
         self._add_segment('end')
 
-    def addComponent(self, glyphName, transformation):
+    def addComponent(
+		self,
```

Comment:
mixing tabs and spaces?
feels weird we add typing only to this method and not the rest.  Maybe for now just rename the parameters if you don't have time to add type hints to all.

Reply:
C&P error ...

Reply:
I've started to add typing annotations to the pens. But I had to stop now, because it gets complicated and even requires some code rewriting to pass the type checks. But at least there is more typing than before now :)

Reply:
Hm, where would it require code rewriting?

Reply:
I needed to add some assertions, otherwise the type checker would complain about variables possibly being None, e.g.:

https://github.com/fonttools/fonttools/pull/2919/files#diff-713c63d7dd345e61c8c06754b31811b60345f57b1b2c70fdf7bb8aaf7227c92aR408-R420